site/content/api/Tenant-API.md

```diff
@@ -252,7 +252,7 @@ The table below provides an overview of the standard members defined for the JSO
 | :------------------------| :-------: | :--------- | :------------ | :---------- |
 | *tenant-id*              | *yes*     | *string*   |               | The ID of the tenant. |
 | *enabled*                | *no*      | *boolean*  | `true`        | If set to `false` the tenant is currently disabled. Protocol adapters MUST NOT allow devices of a disabled tenant to connect and MUST NOT accept data published by such devices.
-| *adapters*   | *yes*     | *array*    |               | A list of configuration options valid for certain adapters only. See [Adapter Configuration Format]({{< relref "#adapter-configuration-format" >}}) for details. **NB** This array must contain at least one element - an empty array is handled as an error. **NB** Only a single entry per *type* is allowed. If multiple entries for the same type are present it is handled as an error.
+| *adapters*               | *no*      | *array*    | empty array   | A list of configuration options valid for certain adapters only. See [Adapter Configuration Format]({{< relref "#adapter-configuration-format" >}}) for details. **NB** Only a single entry per *type* is allowed. If multiple entries for the same type are present it is handled as an error.
```

Comment:
FMPOV the format definition of the array elements can be made a little clearer by rephrasing it to:

A list of configuration options valid for certain adapters only. **The format of a configuration option is described here [Adapter Configuration Format]({{< relref "#adapter-configuration-format" >}}).** **NB** Only a single entry per *type* is allowed. If multiple entries for the same type are present it is handled as an error.

WDYT?

Reply:
Jup I like it more too. Adapted.